src/test/java/study/SetTest.java

```diff
@@ -0,0 +1,64 @@
+package study;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class SetTest {
+    private Set<Integer> numbers;
+
+    @BeforeEach
+    void setUp() {
+        numbers = new HashSet<>();
+        numbers.add(1);
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
+    }
+
+    
+    @Test
+    @DisplayName("Set의 size() 메소드를 활용해 set의 크기 확인")
+    public void compareSetSize() throws Exception {
+        //given
+        //when
+        int size = numbers.size();
+        //then
+        assertThat(size).isEqualTo(3);
```

Comment:
다음과 같이도 작성해볼 수 있을 것 같아요. 다양한 API를 활용해보시는 걸 추천드립니다.
```suggestion
        assertThat(numbers).hasSize(3);
```

Reply:
안녕하세요. 잘 부탁드리겠습니다. : )
혹시, 미션을 하면서 정리하고 싶은데, 제 블로그에 리뷰 등 글을 남겨도 될까요!?

Reply:
넵 괜찮습니다!